### PR TITLE
Add reverifiplus and explorelistings from homepage

### PR DESCRIPTION
**Pull request by Anas-AlRamahi**
### Description
Finished implementing the frontend design for reverifiPlus and exploreListings sections in HomePage

### CODE REVIEW

File: src/components/ExploreListing/explore-listing.styles.js [hunk 1]
```diff
@@ -1,3 +1,196 @@
...
+
+export const FirstImgHeader = styled.h4`
+  font-family: Montserrat;
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 2.4375rem;
+  margin: 0.5625rem 0 0.5625rem 1rem;
+`;
+
+export const FirstImgListings = styled.p`
```
Comment (abukhadraop) [inline src/components/ExploreListing/explore-listing.styles.js:152]: the `p` tag is usually used with paragraphs

**Review by abukhadraop — CHANGES_REQUESTED** [review]
General notes
- please pay attention to the app console errors
- there is no need to suffixe the styled components names with elemnt type (ex `Div`)
- try to use meaningful names for the styled components (ex `SliderDiv`=>`SliderContainer`) 
- keep in mind the lees code to achieve the result is the better, always try to enhance you code after you wrote it

File: src/components/ReverifiPlus/reverifi-plus.styles.js [hunk 2]
```diff
@@ -1,3 +1,97 @@
...
+  margin: 0 auto 0 auto;
+  padding: 2.25rem 1.37rem;
+
+  ${mq.tablet`
+    flex-direction: row;
+    justify-content: center;
+  `}
+`;
+
+export const PlusUpperSection = styled.div`
```
Comment (Mahmoudzaid2994) [inline src/components/ReverifiPlus/reverifi-plus.styles.js:18]: same here and for the rest variables

File: src/components/ReverifiPlus/reverifi-plus.styles.js [hunk 3]
```diff
@@ -1,3 +1,97 @@
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
 import styled from 'styled-components';
 
-export const Title = styled.h3``;
+export const ReverifiPlusContainer = styled.div`
```
Comment (Mahmoudzaid2994) [inline src/components/ReverifiPlus/reverifi-plus.styles.js:5]: please choose a meaningful name for variables

File: src/components/ReverifiPlus/index.jsx [hunk 4]
```diff
@@ -8,9 +24,56 @@ import { Title } from './reverifi-plus.styles';
  */
 function ReverifiPlus() {
   return (
-    <div>
-      <Title>TODO: implement reverifi plus.</Title>
-    </div>
+    <ReverifiPlusContainer>
```
Comment (Mahmoudzaid2994) [inline src/components/ReverifiPlus/index.jsx:27]: for this div the style `justify-content` should be `space-between` and also add margin on left and right

File: src/components/ExploreListing/explore-listing.styles.js [hunk 5]
```diff
@@ -1,3 +1,169 @@
...
+`;
+
+export const SliderContainer = styled.div`
+  & .slick-slider {
+    width: 100%;
+  }
+
+  & .slick-dots {
+    bottom: 0;
+    height: 36.8px;
```
Comment (Mahmoudzaid2994) [inline src/components/ExploreListing/explore-listing.styles.js:23]: Please convert px to rem

File: src/assets/Shape.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="28" viewBox="0 0 32 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Mahmoudzaid2994) [inline src/assets/Shape.svg:1]: please change the names of all SVGs to be kebab case.

File: src/components/ReverifiPlus/index.jsx (hunk 4 above)
Comment (Anas-AlRamahi) [reply src/components/ReverifiPlus/index.jsx:27]: For this one I used a predefined media query, should I commit changes? or is there a better way to implement it?

**Comment by MZeq** [comment]
Can you rename all the images to be brief and descriptive of the shape?

`new-york-city-evening-NYCTG0221(3).jpg` is too long and contains unnecessary sections.
`newyork` or `newyork-city` is a more appropriate and descriptive name.

Same thing for all the SVGs.

`agent-svgrepo-com.svg` the name doesn't describe the shape of the image and `svgrepo-com` is not wanted. 
and `shape` .. etc

File: src/components/ExploreListing/explore-listing.styles.js [hunk 7]
```diff
@@ -1,3 +1,169 @@
...
+  margin: 0 auto 1.1rem auto;
+  text-align: center;
+
+  ${mq.tablet`
+    font-size: 2rem;
+  `}
+`;
+
+export const SliderContainer = styled.div`
+  & .slick-slider {
```
Comment (MZeq) [inline src/components/ExploreListing/explore-listing.styles.js:17]: if we could remove the `&` from the styles as well, wherever it's used.
```suggestion
  .slick-slider {
```

**Comment by Anas-AlRamahi** [comment]
This PR was fixed into a new PR called Rev-002: add reverifiplus and explorelistings from homepage. Last comments were resolved in the new PR as well.

**Pull request closed without merging**